less/dropdowns.less

```diff
@@ -10,7 +10,7 @@
   height: 0;
   margin-left: 2px;
   vertical-align: middle;
-  border-top:   @caret-width-base dashed;
+  border-top:   @caret-width-base solid;
```

Comment:
This makes the next line unnecessary; please delete the now-redundant next line.